pkg/config/scheduler_config.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"time"
+
+	cerrors "github.com/pingcap/tiflow/pkg/errors"
+)
+
+// SchedulerConfig configs TiCDC scheduler.
+type SchedulerConfig struct {
+	HeartbeatTick        int          `toml:"heartbeat-tick" json:"heartbeat-tick"`
```

Comment:
from the field name`HeartbeatTick`, I don't know what it means.

Reply:
Comments added, thanks!